GameStore.API/Controllers/CartController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GameStore.Application.Interfaces;
+using GameStore.Application.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GameStore.API.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/[controller]")]
+public class CartController : ControllerBase
+{
+    private readonly ICartService _cartService;
+
+    public CartController(ICartService cartService)
+    {
+        _cartService = cartService;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<CartModel>> GetCart()
+    {
+        var cart = await _cartService.GetCurrentCartAsync();
+
+        return Ok(cart);
+    }
+
+    [HttpPost("add")]
+    public async Task<ActionResult<CartModel>> AddToCart(CreateOrderItemModel item)
+    {
+        await _cartService.AddToCartAsync(item);
+
+        return Ok();
+    }
+
+    [HttpPut("{itemId}/increase")]
+    public async Task<ActionResult> IncreaseCount(int itemId)
+    {
+        await _cartService.UpdateCountAsync(itemId, 1);
+
+        return Ok();
+    }
+    
+    [HttpPut("{itemId}/decrease")]
```

Comment:
`POST` would be more appropriate for Increase/Decrease endpoints.

`PUT` method is **idempotent**. So if we retry a request multiple times, that should be equivalent to a single request invocation.
`POST` is **NOT idempotent**. So if we retry the request N times, we will end up having N resources with N different URIs created on the server. (It's just example, in our case we're changing state of entity on every request)

https://restfulapi.net/rest-put-vs-post/